### PR TITLE
Bookstore: Style

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I:

Added style to the app.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -9,14 +50,241 @@ header {
...
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 800;
+  color: #868383;
+  margin-bottom: 10px;
+}
+
+.d-flex {
+  display: flex;
+}
```
Comment (Gambit142) [inline src/App.css:217]: - [x] Kindly use this class selector as a `className` for elements that you want their display to be flex. Currently, you are using `display: flex` repetitively in your style sheet and that is not a great way to write CSS declarations. Kindly implement this suggestion 👍

File: src/components/Form.js [hunk 2]
```diff
@@ -36,21 +37,25 @@ const Form = () => {
           value={title}
           onChange={handleChange}
         />
-        <input
+        {/*  <input
           type="text"
           id="author"
           placeholder="Author"
           value={author}
           onChange={handleAuthor}
```
Comment (Gambit142) [inline src/components/Form.js]: - [ ] You should have an input `form` field for `author.` Kindly render the input field for author. 👍 You can render the input for `title`, `author`, the select form control for `category` and the `Add Book` button.

File: src/components/Header.js [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+            <Link to="/">Books</Link>
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </div>
+      <div className="userdiv">
+        <FontAwesomeIcon icon={faUser} />
+      </div>
```
Comment (Gambit142) [inline src/components/Header.js]: - [x] One of the project requirements states that your design should be very identical to that of Zeplin's template provided. Kindly make the Header/Nav Bar look like that in the template.

### Your Design

image

### Zeplin's Template

image

File: src/App.css [hunk 4]
```diff
@@ -9,14 +50,241 @@ header {
...
+
+.pres {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
```
Comment (Gambit142) [inline src/App.css:190]: - [x] You have a lot of repetitions for `colors` and `font-family`. For example, you repeated colors `#fff`, `#0290ff`, `#4386bf`, `#121212` and `#e8e8e8` many times in your style sheet. The same thing was done for `font-family: 'Montserrat', sans-serif;` and `font-family: 'Roboto Slab', serif;`. 

I would advise you to either make them class selectors and use them as `classNames` for elements you are targeting or you make them CSS variables and apply them wherever you wish to in the stylesheets. If you wish to make them CSS variables, you can look up this [article](https://www.w3schools.com/css/css3_variables.asp)

File: src/components/Card.js [hunk 5]
```diff
@@ -5,23 +5,49 @@ import { removeBooksAsync } from '../redux/books/books';
...
+            </span>
+          </div>
+        </div>
+      </div>
+      <div className="progress">
+        <h4>CURRENT CHAPTER</h4>
+        <span className="lesson">Chapter 17</span>
+        <button className="Remove" type="button">Update progress</button>
+      </div>
+    </div>
```
Comment (Gambit142) [inline src/components/Card.js]: - [x] Kindly make the border of the general container vertically align with that of the _**ADD NEW BOOK**_ header for the form control.

### Your Design

alignment

### Zeplin's Template

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @ivaan1992 👋, 

Good job so far!

You did a great job styling the application.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by bimbolabuari — APPROVED** [review]
Hi @ivaan1992,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._t

**Pull request merged**